### PR TITLE
315 bad apirestclient generation

### PR DESCRIPTION
**Pull request by RobertoSngular**

### CODE REVIEW

File: multiapi-engine/src/main/resources/templates/openapi/templateRestClient.ftlh [hunk 1]
```diff
@@ -120,7 +121,7 @@ public class ApiRestClient {
     return dateFormat;
   }
 
-  private static void createDefaultObjectMapper(final DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
+  private static void createDefaultObjectMapper(DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
```
Comment (jemacineiras) [inline multiapi-engine/src/main/resources/templates/openapi/templateRestClient.ftlh:124]: Why you removed the `final`?

File: multiapi-engine/src/main/resources/templates/openapi/templateRestClient.ftlh [hunk 2]
```diff
@@ -191,12 +192,12 @@ public class ApiRestClient {
...
       result = CollectionFormat.CSV.collectionToString(values);
     } else {
-      result = collectionFormat.collectionToString(values)
+      result = collectionFormat.collectionToString(values);
     }
     return result;
   }
 
-  public MultiValueMap<String, String> parameterToMultiValueMap(final CollectionFormat collectionFormat, final String name, final Object value) {
+  public MultiValueMap<String, String> parameterToMultiValueMap(CollectionFormat collectionFormat, final String name, final Object value) {
```
Comment (jemacineiras) [inline multiapi-engine/src/main/resources/templates/openapi/templateRestClient.ftlh:200]: Why you removed the `final`?

File: multiapi-engine/src/test/resources/openapigenerator/testClientPackageWebClientApiGeneration/assets/TestClient.java [hunk 3]
```diff
@@ -120,7 +121,7 @@ public Date parse(final String source, final ParsePosition pos) {
     return dateFormat;
   }
 
-  private static void createDefaultObjectMapper(final DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
+  private static void createDefaultObjectMapper(DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
```
Comment (jemacineiras) [inline multiapi-engine/src/test/resources/openapigenerator/testClientPackageWebClientApiGeneration/assets/TestClient.java:124]: Why you removed the `final`?

File: multiapi-engine/src/test/resources/openapigenerator/testClientPackageWebClientApiGeneration/assets/TestClient.java [hunk 4]
```diff
@@ -191,12 +192,12 @@ public String collectionPathParameterToString(final CollectionFormat collectionF
...
       result = CollectionFormat.CSV.collectionToString(values);
     } else {
-      result = collectionFormat.collectionToString(values)
+      result = collectionFormat.collectionToString(values);
     }
     return result;
   }
 
-  public MultiValueMap<String, String> parameterToMultiValueMap(final CollectionFormat collectionFormat, final String name, final Object value) {
+  public MultiValueMap<String, String> parameterToMultiValueMap(CollectionFormat collectionFormat, final String name, final Object value) {
```
Comment (jemacineiras) [inline multiapi-engine/src/test/resources/openapigenerator/testClientPackageWebClientApiGeneration/assets/TestClient.java:200]: Why you removed the `final`?

File: multiapi-engine/src/test/resources/openapigenerator/testRestClientApiGeneration/assets/client/ApiRestClient.java [hunk 5]
```diff
@@ -120,7 +121,7 @@ public Date parse(final String source, final ParsePosition pos) {
     return dateFormat;
   }
 
-  private static void createDefaultObjectMapper(final DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
+  private static void createDefaultObjectMapper(DateFormat defaultDateFormat, final AbstractJackson2HttpMessageConverter converter) {
```
Comment (jemacineiras) [inline multiapi-engine/src/test/resources/openapigenerator/testRestClientApiGeneration/assets/client/ApiRestClient.java:124]: Why you removed the `final`?

File: multiapi-engine/src/test/resources/openapigenerator/testRestClientApiGeneration/assets/client/ApiRestClient.java [hunk 6]
```diff
@@ -191,12 +192,12 @@ public String collectionPathParameterToString(final CollectionFormat collectionF
...
       result = CollectionFormat.CSV.collectionToString(values);
     } else {
-      result = collectionFormat.collectionToString(values)
+      result = collectionFormat.collectionToString(values);
     }
     return result;
   }
 
-  public MultiValueMap<String, String> parameterToMultiValueMap(final CollectionFormat collectionFormat, final String name, final Object value) {
+  public MultiValueMap<String, String> parameterToMultiValueMap(CollectionFormat collectionFormat, final String name, final Object value) {
```
Comment (jemacineiras) [inline multiapi-engine/src/test/resources/openapigenerator/testRestClientApiGeneration/assets/client/ApiRestClient.java:200]: Why you removed the `final`?

File: scs-multiapi-maven-plugin/pom.xml [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.sngular</groupId>
   <artifactId>scs-multiapi-maven-plugin</artifactId>
-  <version>5.3.1</version>
+  <version>5.3.2</version>
```
Comment (jemacineiras) [inline scs-multiapi-maven-plugin/pom.xml:7]: You are missing change version number in 3 more files...

**Review by jemacineiras — CHANGES_REQUESTED** [review]

File: scs-multiapi-maven-plugin/pom.xml (hunk 7 above)
Comment (RobertoSngular) [reply scs-multiapi-maven-plugin/pom.xml:7]: fixed

**Review by jemacineiras — DISMISSED** [review]

**Pull request merged**